main/background.ts

```diff
@@ -29,6 +33,23 @@ express_app.get('/', (req, res) => {
   `);
 });
 
+express_app.get('/ipaddress', (req, res) => {
+  const networkInterfaces = os.networkInterfaces();
+  let ipv4Address;
+  Object.keys(networkInterfaces).forEach(interfaceName => {
+    networkInterfaces[interfaceName].forEach(networkInterface => {
+      if (networkInterface.family === 'IPv4' && !networkInterface.internal) {
+        ipv4Address = networkInterface.address;
+      }
+    });
+  });
```

Comment:
Perhaps it's worth extracting a separate function findIpv4Address or something like that?